api/Window.json

```diff
@@ -3159,10 +3160,11 @@
             },
             "firefox": {
               "version_added": "1",
-              "version_removed": null
+              "version_removed": "32"
```

Comment:
https://developer.mozilla.org/docs/Web/API/Window/home mentions Firefox 31, so this seems right. If we can also update the Opera support there, then we'll probably find this entry can be removed entirely.

Reply:
Opera support is dubious (if Chrome never supported it, I'd be floored if Opera did), so I'm fine with removing this and redirecting https://developer.mozilla.org/docs/Web/API/Window/home → https://developer.mozilla.org/docs/Web/API/Window

Reply:
I didn't expect it, but turns out Opera 12.16 *did* support `window.home()`. That's so interesting that I sent https://github.com/mdn/browser-compat-data/pull/7551, but I do agree we should delete this entry eventually as it's no longer supported anywhere.